src/routers/submit.router.ts

```diff
@@ -4,6 +4,8 @@ import { env } from "../config";
 const router: Router = Router();
 const { SUBMIT_VALIDATION_URL, CALLBACK_URL } = env;
 router.get('/submit', (_req: Request, res: Response, _next: NextFunction) => {
+    res.header('Access-Control-Allow-Origin', `${env.CHS_URL}`);
+    res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
```

Comment:
Are these still necessary?
Are these related to what you were talking about this morning with the accounts service?
If so I think they are not necessary.